### ISSUE TITLE
Update prettier-eslint to 6.4.2

### ISSUE THREAD

**Issue by OliverJAsh** [esbenp/prettier-vscode#150]
I see that we specify prettier-eslint version ^6.4.0, but upon reinstall I'm getting 6.4.1 every time.

I noticed there was a `package-lock.json` in the extensions folder which is locking it to 6.4.1:

```
~/.vscode/extensions/esbenp.prettier-vscode-0.20.1/package-lock.json
```

Do you know how we can update prettier-eslint to 6.4.2?

**Comment by CiGit** [comment]
It seems I forgot to remove that file before bundling. It may be time to upgrade the project to npm@5.
`npm u prettier-eslint` with npm@5

**Comment by OliverJAsh** [comment]
Can we push a new release out that removes the lockfile?

**Comment by CiGit** [comment]
If you want to update your installation, you can just delete that package-lock. And if you messed it up too much ... reinstall

I'll update prettier-eslint tonight (CET) and publish it. I like your fix in it :+1:

**Comment by CiGit** [comment]
I've just published a version with 6.4.2
